### PR TITLE
fix(image-build): change from emulation to cross-compile

### PR DESCRIPTION
**Pull request by evegufy**
## Description

- change from emulation to cross-compile for building multi-platform images
- also, improve dockerfiles by removing unnecessary base stage and aligning environment variables

## Why

https://github.com/eclipse-tractusx/portal-backend/issues/802
https://docs.docker.com/build/building/multi-platform
https://devblogs.microsoft.com/dotnet/improving-multiplatform-container-support

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own changes
- [x] I have successfully tested my changes

### CODE REVIEW

File: docker/Dockerfile-iam-seeding [hunk 1]
```diff
@@ -34,8 +32,10 @@ COPY /src/keycloak/Keycloak.Seeding /src/keycloak/Keycloak.Seeding
 WORKDIR /src/keycloak/Keycloak.Seeding
 RUN dotnet publish "Keycloak.Seeding.csproj" -c Release -o /app/publish
 
-FROM base AS final
-ENV COMPlus_EnableDiagnostics=0
+FROM mcr.microsoft.com/dotnet/runtime:8.0-alpine
+ENV \
+    COMPlus_EnableDiagnostics=0 \
+    ASPNETCORE_URLS=http://+:8080
```
Comment (Phil91) [inline docker/Dockerfile-iam-seeding:38]: do we need the aspnetcore_urls for the seeding?

File: docker/Dockerfile-portal-migrations [hunk 2]
```diff
@@ -34,8 +32,10 @@ COPY /src/processes/Processes.ProcessIdentity/ /src/processes/Processes.ProcessI
 WORKDIR /src/portalbackend/PortalBackend.Migrations
 RUN dotnet publish "PortalBackend.Migrations.csproj" -c Release -o /migrations/publish
 
-FROM base AS final
-ENV COMPlus_EnableDiagnostics=0
+FROM mcr.microsoft.com/dotnet/runtime:8.0-alpine
+ENV \
+    COMPlus_EnableDiagnostics=0 \
+    ASPNETCORE_URLS=http://+:8080
```
Comment (Phil91) [inline docker/Dockerfile-portal-migrations:38]: I think we can remove the aspnetcore_urls for the migration

File: docker/Dockerfile-processes-worker [hunk 3]
```diff
@@ -17,18 +17,18 @@
...
 RUN dotnet restore "src/processes/Processes.Worker/Processes.Worker.csproj"
 WORKDIR /src/processes/Processes.Worker
 RUN dotnet publish "Processes.Worker.csproj" -c Release -o /app/publish
 
-FROM base AS final
-ENV COMPlus_EnableDiagnostics=0
+FROM mcr.microsoft.com/dotnet/runtime:8.0-alpine
+ENV \
+    COMPlus_EnableDiagnostics=0 \
+    ASPNETCORE_URLS=http://+:8080
```
Comment (Phil91) [inline docker/Dockerfile-processes-worker:31]: I think we can remove the aspnetcore_urls for the process_worker

File: docker/Dockerfile-provisioning-migrations [hunk 4]
```diff
@@ -31,8 +29,10 @@ COPY /src/framework/Framework.Logging /src/framework/Framework.Logging
 WORKDIR /src/provisioning/Provisioning.Migrations
 RUN dotnet publish "Provisioning.Migrations.csproj" -c Release -o /migrations/publish
 
-FROM base AS final
-ENV COMPlus_EnableDiagnostics=0
+FROM mcr.microsoft.com/dotnet/runtime:8.0-alpine
+ENV \
+    COMPlus_EnableDiagnostics=0 \
+    ASPNETCORE_URLS=http://+:8080
```
Comment (Phil91) [inline docker/Dockerfile-provisioning-migrations:35]: I think we can remove the aspnetcore_urls for the migration

**Review by Phil91 — CHANGES_REQUESTED** [review]

File: docker/Dockerfile-iam-seeding (hunk 1 above)
Comment (evegufy) [reply docker/Dockerfile-iam-seeding]: changed

**Review by Phil91 — APPROVED** [review]

**Pull request merged**